ui/src/app/machines/views/MachineDetails/MachineNetwork/NetworkTable/NetworkTable.tsx

```diff
@@ -500,6 +542,11 @@ const NetworkTable = ({ systemId }: Props): JSX.Element => {
         {
           content: (
             <>
+              <GroupCheckbox
+                items={nicIds}
```

Comment:
Maybe this should include the disabled prop if networking is disabled?

Reply:
Oops yes it needs that!